sdk/src/manifest.rs

```diff
@@ -1279,6 +1320,7 @@ impl Manifest {
     /// This should only be used for special cases, such as converting an embedded manifest
     /// to a cloud manifest
     #[cfg(feature = "file_io")]
```

Comment:
Not a blocker, but I think these could be coalesced into:

```
#[cfg(all(feature = "file_io", feature = "v1_api"))]
```

Reply:
Since I'm planning to rip out the v1 api entirely in the not to distant future, I've opted to not overly integrate this. But point taken.